src/main/java/net/journey/entity/projectile/EntitySentryKingGrenade.java

```diff
@@ -19,6 +19,7 @@
 import java.util.List;
 
 public class EntitySentryKingGrenade extends EntityTippedArrow implements IProjectile {
```

Comment:
Are all these constructors really needed?

Reply:
Some of them could probably be removed. I just made variants for each existing constructor.